api/scripts/reprocess_metadata.py

```diff
@@ -0,0 +1,51 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+import asyncio
+import base64
+import json
+import logging
+
+import asyncclick as click
+from databases.core import Database
+
+from notd.store.saver import Saver
+from notd.store.schema import TokenMetadataTable
+from notd.store.schema_conversions import token_metadata_from_row
+
+
+@click.command()
+@click.option('-s', '--start-id-number', 'startIDNumber', required=True, type=int)
+@click.option('-e', '--end-block-number', 'endIDNumber', required=True, type=int)
+@click.option('-b', '--batch-size', 'batchSize', required=False, type=int, default=100)
+async def fix_metadata(startIDNumber: int, endIDNumber: int, batchSize: int):
```

Comment:
make this script and function name specific for base64_metadata please, we will probably have more of these in the future

Reply:
also `startIDNumber` -> `startId` and same for end